app/javascript/react/components/ShopIndexPage.js

```diff
@@ -36,8 +37,16 @@ const ShopIndexPage = () => {
 
   return (
     <div>
-    <h1>All Bubble Tea Shops</h1>
-      {shopTiles}
+      <div>
+        <h1>All Bubble Tea Shops</h1>
+        {shopTiles}
+      </div>
+
+      <Link to={`/shops/new`}>
```

Comment:
this path is not dynamic so it doesn't need to be wrapped in curly brackets with back ticks